### PR TITLE
vm_manager: add Pacemaker remote node support

### PR DESCRIPTION
**Pull request by dupremathieu**
Add support for the VirtualDomain remote_node meta parameter. This option can be set to inform Pacemaker that the VM have a pacemaker-remote service running on it.

pacemaker-remote offer a better monitoring and allow manage ressources inside the VM.

### CODE REVIEW

**Review by Revalioli — COMMENTED** [review]
### Commit message
- "inform Pacemaker that the VM have a" --> "inform Pacemaker that the VM **has** a"
- "pacemaker-remote offer a better monitoring and allow manage ressources" --> "pacemaker-remote **offers** a better monitoring and **allows ressources management**"
- "Create two now commands" --> "Create two **new** commands"

### Adding a remote
With your commit you add a remote to a VM by adding meta after meta with `crm resource meta`. Between the moment the `remote-node` meta is added and `remote-addr` is added, the daemon wouldn't know of to connect to the remote, could it break the configuration?

File: vm_manager/helpers/pacemaker.py [hunk 1]
```diff
@@ -222,6 +223,14 @@ def add_vm(self, vm_options):
             "timeout='" + vm_options.get("monitor_timeout", "60") + "'",
             "interval='" + vm_options.get("monitor_interval", "10") + "'",
         ]
+        if vm_options.get("pacemaker_remote"):
+            args += ["meta", "remote-node=" + vm_options["pacemaker_remote"]]
+            if vm_options.get("pacemaker_remote_addr"):
+                args += ["remote-addr='" + vm_options["pacemaker_remote_addr"]]
```
Comment (Revalioli) [inline vm_manager/helpers/pacemaker.py:236]: Missing `+ "'"` at the end of string
Do it also for the following options

File: vm_manager/helpers/pacemaker.py [hunk 2]
```diff
@@ -345,15 +354,7 @@ def add_meta(self, key, value):
...
-        args = [
-            "crm",
-            "resource",
-            "meta",
-            self._resource,
-            "set",
-            key,
-            value
-        ]
+        args = ["crm", "resource", "meta", self._resource, "set", key, value]
```
Comment (Revalioli) [inline vm_manager/helpers/pacemaker.py:357]: Don't change the way you construct the command just here, or change it everywhere

**Review by Revalioli — COMMENTED** [review]
## `vm_manager: add Pacemaker remote node support`

- "code style" or "coding style" is better for me than the word "cosmetic"

File: vm_manager/helpers/pacemaker.py (hunk 2 above)
Comment (dupremathieu) [reply vm_manager/helpers/pacemaker.py:366]: This line was over 79 caracters.

**Review by Revalioli — APPROVED** [review]

**Pull request merged**